drivers/include/adxl345.h

```diff
@@ -120,29 +130,31 @@ typedef struct {
     uint8_t time_inact;   /**< Inactivity time */
     uint8_t thres_ff;     /**< Free-fall threshold */
     uint8_t time_ff;      /**< Time threshold */
-    uint8_t act_inact;    /**< Axis enable control for activity and inactivity detection */
+    uint8_t act_inact;    /**< Enable ctrl for activity/inactivity detection */
     uint8_t tap_axes;     /**< Axis control for single tap/double tap */
 } adxl345_interrupt_t;
 
 /**
  * @brief Configuration struct for the ADXL345 sensor
  */
 typedef struct {
-    i2c_t i2c;                     /**< I2C device which is used */
-    uint8_t addr;                  /**< I2C address */
-    adxl345_interrupt_t interrupt; /**< Interrupts configuration  */
-    uint8_t offset[3];             /**< offset axis */
-    uint8_t range;                 /**< Sensitivity configuration */
-    uint8_t rate;                  /**< Configured sample rate for accel  */
-    uint8_t full_res;              /**< Resolution bit */
-    uint8_t scale_factor;          /**< Scale factor for converting value to mg */
```

Comment:
`full_res` is not deducted from range so it will belong to `adxl345_params_t`
I will move each part to right struct